### ISSUE TITLE
Link to other contributor guides

### ISSUE THREAD

**Issue by pmkovar** [redhat-documentation/doc-guidelines#5]
The index page for this repo could also include links to RH upstream docs contributor guides other than those included in this repo.

Alternatively, add those links to a section in https://github.com/redhat-documentation/community-collaboration-guide.

**Comment by pmkovar** [comment]
cc @ncbaratta

**Comment by rkratky** [comment]
I agree. Now that https://redhat-documentation.github.io/ is featured on https://redhatofficial.github.io/, we should make our page a general intro to RH docs. So, not only link to other doc guidelines but also perhaps spiff it up a bit to have some RH branding.

**Comment by rkratky** [comment]
Oops, didn't mean to close... those buttons are too 'close' to each other.

**Comment by pmkovar** [comment]
Maybe even reuse parts of the overall design from https://redhat-documentation.github.io/ (https://github.com/RedHatOfficial/RedHatOfficial.github.io)?

**Comment by rkratky** [comment]
> Maybe even reuse parts of the overall design

Good idea.
